mappings/mapper.go

```diff
@@ -9,16 +9,19 @@ import (
 
 	"github.com/jtblin/kube2iam"
 	"github.com/jtblin/kube2iam/iam"
+	"regexp"
```

Comment:
Probably these two packages go in the first group with `fmt` as [they're standard library packages](https://github.com/golang/go/wiki/CodeReviewComments#imports).